### ISSUE TITLE
Cannot make the validated field required

### ISSUE THREAD

**Issue by scottbeacher** [kamronbatman/joi-password-complexity#3]
I'm trying to chain the usual Joi parameters on the end to make a parameter required and it causes an error.

```
 password: new PasswordComplexity({ min: 8, lowerCase: 1, numeric: 1, symbol: 1 }).required()
```

It throws:

```
    "message": "Cannot read property 'min' of undefined",
    "stack": "TypeError: Cannot read property 'min' of undefined\n    at PasswordComplexity._base (c:\\project\\node_modules\\joi-password-complexity\\lib\\index.js:35:31)\n    at PasswordComplexity._validate (c:\\project\\node_modules\\joi\\lib\\types\\any\\index.js:590:37)\n    at _base (c:\\project\\node_modules\\joi\\lib\\types\\object\\index.js:157:45)\n 
....
````

Works if I don't add the `required()` to the end but then joi doesn't enforce having that field. I get the same error if I try to add `.options()` to the end to change the default language for the error message since the default isn't very descriptive (why not have it say what the rules are?)

For some reason when adding anything to the end it causes `this._options` to be undefined.

**Comment by kamronbatman** [comment]
I'll look into this!

**Comment by kamronbatman** [comment]
Fixed this in v2.0.1, please update.
